### DISCUSSION TITLE
Is bulk update possible?

### DISCUSSION THREAD

**Discussion by davycardinaal** [brendon/positioning#22] [Q&A]
@brendon is it possible to bulk update a certain list?

Given we have a frontend that supports drag-and-drop sorting, we want to push the new state to the server in one request.

**Comment by brendon** [comment]
Hi @davycardinaal, that's covered in the readme. Rather than supplying a large list of updates, just tell `positioning` what item the selected item is `before` or `after`. In my experience `before` works better. The syntax is `position: { before: 5 }` to positioning before the item with the `id` of `5`. You can then just send each reorder operation in turn as it occurs.

If that's not possible then you'd need to just do a bulk set of `update_all`s to avoid all the callbacks.
